tests/unit/amo/components/TestSponsoredAddonsShelf.js

```diff
@@ -216,6 +262,82 @@ describe(__filename, () => {
 
       sinon.assert.notCalled(dispatchSpy);
     });
+
+    it('sends a ping for the impression on mount', () => {
```

Comment:
is it "ping" or beacon?

Reply:
I'm calling it a ping as a colloquial term. You're right, it's a beacon. I'll change it.